srcpkgs/rawtherapee/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'rawtherapee'
 pkgname=rawtherapee
 version=5.9
-revision=2
+revision=3
 build_style=cmake
 configure_args="-DCACHE_NAME_SUFFIX=\"\""
 hostmakedepends="pkg-config"
 makedepends="fftw-devel gtkmm-devel lensfun-devel
  libcanberra-devel libgomp-devel libiptcdata-devel librsvg-devel"
 depends="desktop-file-utils hicolor-icon-theme"
 short_desc="Free RAW converter and digital photo processing software"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Daniel Martinez <danielmartinez@airmail.cc>"
```

Comment:
As noted in your other PR, please delay adoption of this package until you've established a history of package maintenance.